### DISCUSSION TITLE
option to download expressions from SPDX.org's github license list

### DISCUSSION THREAD

**Discussion by glen-nicol** [sensslen/nuget-license#121] [Ideas]
As mentioned by the [nuget docs](https://learn.microsoft.com/en-us/nuget/nuget-org/licenses.nuget.org#protocol) the licenses.nuget.org URLs are meant for viewing in a browser. It would be very helpful to be able to get the raw license template instead of a full HTML page so that I can incorporate these into a larger aggregate of all the 3rd party dependencies of an application.

Would a PR with options to pull from the [spdx.org github](https://github.com/spdx/license-list-data/blob/main/README.md) be welcomed to do this? They have the raw templates in txt form as well as JSON data files of all the license options in a single JSON array file.

**Comment by sensslen** [comment]
@glen-nicol pull requests are always welcomed.

**Comment by sensslen** [comment]
For SPDX licenses we now link a package that contains all licenses easily readable as string
